package.json

```diff
@@ -75,5 +75,5 @@
     "typedoc": "^0.23.20",
     "typescript": "4.8.2"
   },
-  "version": "0.0.0"
+  "version": "0.0.1"
```

Comment:
Please generate docs as well into this commit, it will make a new doc page for 0.0.1